cpp/search/search.h

```diff
@@ -507,7 +507,7 @@ struct Search {
   int findTopNPolicy(const SearchNode* node, int n, PolicySortEntry* sortedPolicyBuf) const;
 
   std::shared_ptr<NNOutput>* maybeAddPolicyNoiseAndTemp(SearchThread& thread, bool isRoot, NNOutput* oldNNOutput) const;
-
+  bool isSymDupLoc[Board::MAX_ARR_SIZE];
```

Comment:
Done! But I got a small question. Besides coding style, why we use struct instead of class? and member data field are exposed to public?

Reply:
There is not a great reason except for legacy purposes, since Search is a pretty old part of the code and it was that way originally. A small number of fields are accessed very frequently in a read-only way, almost all of the remaining fields are never accessed. We might want to clean it up eventually.